cmd/stack.go

```diff
@@ -198,19 +201,34 @@ func setupStackCommand() *cobraext.Command {
 				return cobraext.FlagParsingError(err, cobraext.ProfileFlagName)
 			}
 
+			shellName, err := cmd.Flags().GetString(cobraext.ShellInitShellFlagName)
+			if err != nil {
+				return cobraext.FlagParsingError(err, cobraext.ShellInitShellFlagName)
+			}
+
+			if shellName == "detect" {
+				shellName, err = detectShell()
+				if err != nil {
+					return fmt.Errorf("cannot detect parent shell from current process: %w", err)
+				}
+				fmt.Fprintf(cmd.OutOrStderr(), "detected shell: %s\n", shellName)
```

Comment:
Leftover? It is expected that `elastic-package stack shellinit` output can be evaluated.

Reply:
That string is sent to `stderr`, so should not affect evaluation:

```
$ ./elastic-package stack shellinit | source
detected shell: fish
$ printenv | ag ELASTIC_PACKAGE_
ELASTIC_PACKAGE_ELASTICSEARCH_HOST=https://127.0.0.1:9200
ELASTIC_PACKAGE_KIBANA_HOST=https://127.0.0.1:5601
ELASTIC_PACKAGE_ELASTICSEARCH_PASSWORD=changeme
ELASTIC_PACKAGE_ELASTICSEARCH_USERNAME=elastic
ELASTIC_PACKAGE_CA_CERT=/home/endorama/.elastic-package/profiles/default/certs/ca-cert.pem
```

Is this acceptable?

Reply:
Ah ok, then I guess that this is fine, we print other messages and warnings to stderr.

As nitpicking:
```suggestion
				fmt.Fprintf(cmd.OutOrStderr(), "Detected shell: %s\n", shellName)
```